server/GraknServer.java

```diff
@@ -66,10 +68,11 @@
 
     private final Grakn grakn;
     private final Server server;
-    private final ServerOptions options;
+    private final ServerCommand.Start options;
     private final GraknRPCService graknRPCService;
+    private final MigratorRPCService migratorRPCService;
 
-    private GraknServer(ServerOptions options) throws IOException {
+    private GraknServer(ServerCommand.Start options) throws IOException {
```

Comment:
Let's rename `options` (and the matching private field) to `command`.

Reply:
done.